src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -444,5 +460,182 @@ internal static bool ContainsPathComponent(string path, string component, bool i
 
             return false;
         }
+
+        /// <summary>
+        /// Gets a path relative to a directory.
+        /// </summary>
+        public static string GetRelativePath(string directory, string fullPath)
```

Comment:
@tannergooding We can't easily copy it. The CoreFX implementation has a bunch of dependencies on internal helpers. A copy would be possible but certainly not easy.